### DISCUSSION TITLE
Is it possible to apply colormap ?

### DISCUSSION THREAD

**Discussion by lchauvin** [Kitware/VolView#596] [Q&A]
Hello,

I was wondering if it's possible to apply colormaps to the images (in 2D views) ?

I found it in the rendering module, but I can't find it for 2D views.

Thank you.

**Comment by PaulHax** [comment]
If you "layer" an image on another, VolView picks and applies a colormap.

**Reply by lchauvin** [reply]
I'm more talking about LUTs

**Reply by PaulHax** [reply]
Right, layer's are not just grayscale:
https://github.com/Kitware/VolView/blob/9e650be5cd0915250ac5237d104898ce31612862/src/components/vtk/VtkLayerSliceRepresentation.vue#L68-L85

https://github.com/Kitware/VolView/blob/9e650be5cd0915250ac5237d104898ce31612862/src/store/view-configs/layers.ts#L104C9-L120

**Reply by lchauvin** [reply]
But how could I change this ? Is there a module to control this ?

**Reply by PaulHax** [reply]
There is no GUI.  Just wanted to point out how you could approach implementing colormap setting for the base slice views.

**Reply by lchauvin** [reply]
I see. Thanks :)
